LEAF_Nexus/api/controllers/GroupController.php

```diff
@@ -67,7 +67,17 @@ public function delete($act)
         $this->index['DELETE']->register('group/[digit]/tag', function ($args) use ($group) {
             try
             {
-                $group->deleteTag((int)$args[0], $group->sanitizeInput($_GET['tag']));
+                return $group->deleteTag((int)$args[0], $_GET['tag'], $_GET['requiresAdmin']);
```

Comment:
Not sure the "requiresAdmin" is needed anymore and tag GET should be run through the sanitizeInput function.

Reply:
Good spot thank you